tests/helpers/Transactions/createTransaction.test.js

```diff
@@ -0,0 +1,35 @@
+const createAppointment = require("../../../helpers/Appointments/createAppointment");
+const createTransaction = require("../../../helpers/Transactions/createTransaction");
+const createUser = require("../../../helpers/Users/createUser");
+const createCourse = require("../../../helpers/Courses/createCourse");
+const { createTables, wipeDBTables } = require("../../../db/databaseHelpers");
+
+beforeAll(async() =>{
+  await createTables();
+  await createUser("1", "txiong@", "password", "student");
+  await createUser("2", "tutor@", "password", "tutor");
+  await createCourse("2", "History", 12, 334);
+  await createAppointment("1", "pending", new Date(), "campus", "2", "1", "2");
+});
+afterAll(async() =>{
+  await wipeDBTables();
+});
+describe("creating a transaction in model", () =>{
+  it("should return true", async() =>{
+    const transaction = await createTransaction("1", "almost paid", 133234, new Date(), "1");
```

Comment:
There is no `almost paid`. Are only options are going to be `paid`, `pending`, or `not paid`.

Probably good to define these as constants somewhere.

Reply:
0e6c45acbb61df9a0b9f60e03e68ad21a4fc904c